contents/tutorials/stripe-reports.md

```diff
@@ -0,0 +1,185 @@
+---
+title: How to set up Stripe reports
+date: 2024-06-21
+author:
+ - ian-vanagas
+tags:
+ - data warehouse
+---
+
+For your startup to become a real business, it needs to make money. Stripe is the most popular choice for helping you do this. Creating and analyzing reports for your Stripe data helps you understand how you are making money and how you can improve.
+
+This tutorial shows you how to sync your Stripe data to PostHog and then create a report of insights like customer count, gross revenue, recurring revenue, revenue churn, and more.
+
+## Linking Stripe data to PostHog
+
+To start, you need both a Stripe and PostHog account. Once have those, head to PostHog's [data warehouse tab](https://us.posthog.com/data-warehouse) and click **Link source**. Choose the Stripe option by clicking **Link**, and then enter your account ID and client secret, and press **Next**. Keep all tables selected and click **Import**. 
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/link_light_d827d9f83f.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/link_dark_6622bd98c2.png"
+  alt="Linking Stripe Account"
+  classes="rounded"
+/>
+
+Once done, PostHog will automatically pull and format your Stripe data for querying. You can adjust the sync frequency, see the last successful run, and more in [data warehouse settings](https://us.posthog.com/data-warehouse/settings/managed).
+
+## Creating insights for your Stripe report
+
+Now that your Stripe data is synced into PostHog, you can use it to create insights for your report. Each of these requires you to create a [new insight in the product analytics tab](https://us.posthog.com/project/insights/new).
+
+### Customer count
+
+To start, we create a trend of customer count over time. 
+
+On the trends tab, change the aggregation to **Unique users**, and then click the data series, go to the **Data Warehouse** tab, hover over the `stripe_customer` table, and click **Select**. You might want to change the **Distinct ID field** from `id` to `email` as Stripe can give multiple `id` values to the same user email.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/customer_light_944920946a.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/customer_dark_e18fc2d09c.png"
+  alt="Customer Count Insights"
+  classes="rounded"
+/>
+
+This can also be done for any of the other Stripe data like charges, subscriptions, and invoices. You can also add filters based on property values like `created_at`, `email`, `status`, and more.
+
+### Gross revenue
+
+Next, we can get our gross revenue by doing a similar process and selecting `stripe_charge`. For aggregation, we want **Property value sum** and then choose `amount`. We also want to filter out failed charges by clicking the filter button, selecting the `status` property, and making sure it doesn't equal `failed`.
```

Comment:
I dont see the stripe_charge table in my warehouse 🤔 Maybe this is something to do with the fact I created a restricted key with read-only values for everything? I dont think so though.

Edit: I see it now. It was just taking time to sync all the data. We should make a note of this for users, since they're likely to run into this issue too

<img width="258" alt="Screenshot 2024-06-24 at 8 55 38 AM" src="https://github.com/PostHog/posthog.com/assets/7090054/0d42b61d-c0fd-4369-9319-79ac447f3f4f">